api/response/msgpack.go

```diff
@@ -0,0 +1,38 @@
+package response
+
+import (
+	"github.com/tinylib/msgp/msgp"
+)
+
+type Msgpack struct {
```

Comment:
so it seems the whole and only reason we have this new type, and don't just keep using the old Msgp type for this new use case, is the slightly different header? otherwise they seem interchangeable. correct?

Reply:
Yes, that is the only difference.